### DISCUSSION TITLE
question about autoDJ queue length

### DISCUSSION THREAD

**Discussion by silvaalex2021** [AzuraCast/AzuraCast#4053] [Q&A]
since there is no documentation on this topic, i would like to ask directly here to get some advice on how to set up the azuracast autoDJ queue.

In the "autoDJ" tab of the station profile, it is possible to configure the AutoDJ queue Length ,. The default value is 3, which I have used most of the time. If I change this to, say, 10, 15, 20 or 25, will I have an advantage or will I have problems?

I have seen, when looking through the Issues here, that many guys put the AutoDJ queue length at 20, for example. is there any technical reason for doing it?

thanks in advance for your advice!

**Comment by Liam024** [comment]
So for me I often had scheduling problems with playlists, ad triggers and other things as soon as my value there is higher than 3. Otherwise I can only recommend: Test it out! But maybe the others can tell you more than I can.
